src/User32.Desktop/User32.Helpers.cs

```diff
@@ -37,9 +37,20 @@ public static SafeDCHandle GetDC(IntPtr hWnd)
         /// <exception cref="Win32Exception">Thrown when an error occurs.</exception>
         public static unsafe string GetClassName(IntPtr hWnd)
         {
-            const int bufferSize = 256;
-            char* className = stackalloc char[bufferSize]; // max class name length
-            int count = GetClassName(hWnd, className, bufferSize);
+            return GetClassName(hWnd, 256);
```

Comment:
Any particular reason you avoided simply using a default parameter in favor of a distinct overload? As a policy on the project, I don't think we avoid default parameters.